Factory/RequestFactory.php

```diff
@@ -50,6 +52,12 @@ public function createOne(array $tokens, $apiUrl, array $options) {
         $query   = HttpQuery::create($tokens);
         $payload = $method === HttpMethods::GET || $method === HttpMethods::DELETE ? null : Payload::create($tokens);
 
+        $options['CURLOPT_HTTPHEADER'] = array_merge(
```

Comment:
Oh yes sorry it was the tokens array I mentioned and not the options array. Was my mistake